pkg/state/helmx.go

```diff
@@ -54,7 +54,7 @@ func formatLabels(labels map[string]string) string {
 
 // append labels flags to helm flags, starting from helm v3.13.0
 func (st *HelmState) appendLabelsFlags(flags []string, helm helmexec.Interface, release *ReleaseSpec, syncReleaseLabels bool) []string {
-	if helm.IsVersionAtLeast("3.13.0") && (syncReleaseLabels || release.SyncReleaseLabels) {
+	if helm.IsVersionAtLeast("3.13.0") && (syncReleaseLabels || st.HelmDefaults.SyncReleaseLabels || release.SyncReleaseLabels) {
```

Comment:
```suggestion
if helm.IsVersionAtLeast("3.13.0") && (syncReleaseLabels || release.SyncReleaseLabels || st.HelmDefaults.SyncReleaseLabels) {
```

Even though this probably should have been a separate MR i would probably add `st.HelmDefaults.SyncReleaseLabels` to the end of the or chain IF it should act as fallback to allow previous behavior to work as expected.